app/views/user_mailer/error_occurred.html.haml

```diff
@@ -1,19 +1,25 @@
 .col-lg-6
   %hr
-  %h4 #{@message.content}
+  %p An error has just occurred. View full details at:
+  %a http://tralala-url.com/ceva/undeva
+  %p{:style => 'color: gray'} ID
+  %p #{@issue.event_id}
   %hr
-  %p An error has just occurred.
-  %strong Error message:
-  %p #{@message.content}
-  %strong Where:
-  %p
-    - JSON.parse(@message.issue.description).each do |file|
-      #{file["filename"]}, line no: #{file["lineno"]}, colno: #{file["colno"]} 
-      %br
-  %strong Backtrace Summary:
-  %p
-    - JSON.parse(@message.issue.data).each do |backtrace|
-      - backtrace.each do |type, content|
-        #{content}
-        %br
-        %br
\ No newline at end of file
+  %p{:style => 'color: gray'} Error type
+  %p #{@issue.get_interfaces(:exception)._data[:values].first._data[:type]}
+  %p{:style => 'color: gray'} Error message
+  %p #{@issue.get_interfaces(:exception)._data[:values].first._data[:value]}
+  %p{:style => 'color: gray'} When
+  %p #{@issue.created_at}
+  %p{:style => 'color: gray'} Where
+  %p #{@issue.stacktrace_frames.first._data[:abs_path]}
+  %p{:style => 'color: gray'} File
+  %p #{@issue.stacktrace_frames.first.get_culprit_string(with_lineno: true)}
```

Comment:
Considering you ar using the `@issue.stacktrace_frames.first` everywhere I would make a variable and use that here. So you don't call the stacktrace_frames method each time.